dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard_INIT.mo

```diff
@@ -0,0 +1,53 @@
+within Dynawo.Electrical.StaticVarCompensators;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model SVarCStandard_INIT "Initialization for standard static var compensator model"
+
+  import Dynawo.Electrical.Sources.InjectorBG_INIT;
+  import Dynawo.Electrical.SystemBase;
+
+  extends AdditionalIcons.Init;
+
+  parameter Types.VoltageModulePu U0Pu  "Start value of voltage amplitude at injector terminal in p.u (base UNom)";
+  parameter Types.Angle UPhase0  "Start value of voltage angle at injector terminal (in rad)";
+  parameter Types.ActivePowerPu P0Pu  "Start value of active power in p.u (base SnRef) (receptor convention)";
+  parameter Types.ReactivePowerPu Q0Pu  "Start value of reactive power in p.u (base SnRef) (receptor convention)";
+
+  parameter Types.VoltageModule UNom "Static var compensator nominal voltage in kV";
+  parameter Types.ApparentPowerModule SNom "Static var compensator nominal apparent power in MVA";
+  parameter Types.PerUnit Lambda "Statism of the regulation law URefPu = UPu - Lambda*QPu";
+
+  Types.ComplexVoltagePu u0Pu "Start value of complex voltage at injector terminal in p.u (base UNom)";
+  Types.ComplexApparentPowerPu s0Pu "Start value of complex apparent power in p.u (base SnRef) (receptor convention)";
+  flow Types.ComplexCurrentPu i0Pu "Start value of complex current at load terminal in p.u (base UNom, SnRef) (receptor convention)";
+
+  Types.PerUnit G0Pu "Start value of the conductance in p.u (base SNom)";
+  Types.PerUnit B0Pu "Start value of the susceptance in p.u (base SNom)";
+  Types.VoltageModule URef0  "Start value of voltage reference in kV";
+
+  InjectorBG_INIT injector(SNom = SNom, U0Pu = U0Pu, UPhase0 = UPhase0, P0Pu = P0Pu, Q0Pu = Q0Pu);
```

Comment:
This is the first time that we implement such a thing in an init model. Is there a specific reason behind this choice? Do you need this to mimic the behavior that we had with the initConnect in the preassembled model?

Reply:
If we want to use the InjectorBG_INIT model, I think we do. We had the same issue with the old HVDC standard model and we ended up with this way of propagating the init parameters.

Reply:
ok!